x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1492,9 +1637,11 @@ public String toString() {
          * "elastic" username.
          */
         AuditEventMetaInfo(Optional<User> user, Optional<String> realm, Optional<AuthorizationInfo> authorizationInfo,
-                           Optional<String[]> indices) {
+                           Optional<String[]> indices, Optional<String> action, Authentication authentication,
+                           TransportRequest transportRequest) {
```

Comment:
Nit: if this portion of the code uses `Optional` it's muddying to add arguments that use `null`.